xml/Microsoft.Extensions.Hosting/HostBuilder.xml

```diff
@@ -104,9 +104,11 @@
       <Docs>
         <param name="configureDelegate">The delegate for configuring the <see cref="T:Microsoft.Extensions.Configuration.IConfigurationBuilder" /> that will be used
             to construct the <see cref="T:Microsoft.Extensions.Configuration.IConfiguration" /> for the application.</param>
-        <summary>Sets up the configuration for the remainder of the build process and application. This can be called multiple times and
-            the results will be additive. The results will be available at <see cref="P:Microsoft.Extensions.Hosting.HostBuilderContext.Configuration" /> for
-            subsequent operations, as well as in <see cref="P:Microsoft.Extensions.Hosting.IHost.Services" />.</summary>
+        <summary>Sets up the configuration for the remainder of the build process and application.
```

Comment:
What is a `DI container`? Any chance you can expand that acronym?

Reply:
DI, dependency injection container, perhaps it's ok to keep `DI container` as-is. there are other instances of uses for DI in docs:

https://github.com/dotnet/dotnet-api-docs/search?q=di+container